test/endtoend.test.ts

```diff
@@ -0,0 +1,140 @@
+import request from 'supertest';
+import server from '../src';
+import { ETaskStatus } from '../src/models/share';
+import { Task } from '../src/models/task.model';
+import { sequelize } from '../src/services/database';
+
+const testTasks = [
+  { id: '180aed45-68cf-4e27-88e0-929e6ccc1400',
+    task: 'Buy eggs',
+    status: ETaskStatus.Incomplete},
+  { id: '5daaa7f0-c4ad-4731-8cca-e370cc07e774',
+    task: 'Buy milk',
+    status: ETaskStatus.Complete},
+  { id: '120ba7bf-1cef-4b0c-8bdd-7cd5a13ad888',
+    task: 'Buy cheese',
+    status: ETaskStatus.Incomplete},
+  { id: '2431c8c0-c3be-42eb-8b19-4c3c33d72789',
+    task: 'Buy bacon',
+    status: ETaskStatus.Incomplete},
+];
+
+const instantiateDB = async () => {
+  await Task.sync();
+  await Task.destroy({
+      truncate: true
+  });
+  await Task.bulkCreate(testTasks);
+}
+
+beforeEach(async ()=>{
```

Comment:
This is not a request. I think it will be better to place both beforeEach and afterAll blocks inside the first `describe` block